src/bin/cql-stress-cassandra-stress/settings/command/user.rs

```diff
@@ -157,9 +164,10 @@ impl UserParams {
             table_definition,
             mut queries,
         } = handles.profile.get().unwrap();
-        let queries_ratio = handles.ratio.get().unwrap();
+        let mut queries_ratio = handles.ratio.get().unwrap();
         let clustering: Arc<dyn DistributionFactory> = handles.clustering.get().unwrap().into();
 
+        let insert_operation_weight = queries_ratio.remove(PREDEFINED_INSERT_OPERATION);
         let queries_payload = queries_ratio
             .into_iter()
             .map(
```

Comment:
I don't understand why this particular (key, value) pair is removed from the map here, while the remaining ones are removed later. Could you provide an example CLI input that shows examples of both this particular pair and different pairs?

Reply:
Having read the strings later in this commit I think I understand. Can you add some comments explaining this specific treatment of `PREDEFINED_INSERT_OPERATION`?